packages/core/addon/models/csv.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Can we move these fragments under `packages/core/addon/models/fragments/delivery-formats/`

Reply:
no because it can't find the schema. even if you register it under a different 'name'. I tried